backend/src/test/java/com/bakdata/conquery/models/datasets/concepts/tree/MatchingStatsTests.java

```diff
@@ -0,0 +1,55 @@
+package com.bakdata.conquery.models.datasets.concepts.tree;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+import com.bakdata.conquery.models.datasets.Column;
+import com.bakdata.conquery.models.datasets.Dataset;
+import com.bakdata.conquery.models.datasets.Table;
+import com.bakdata.conquery.models.datasets.concepts.MatchingStats;
+import com.bakdata.conquery.models.identifiable.ids.specific.DatasetId;
+import com.bakdata.conquery.models.identifiable.ids.specific.WorkerId;
+import it.unimi.dsi.fastutil.ints.IntOpenHashSet;
+import it.unimi.dsi.fastutil.ints.IntSet;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+
+public class MatchingStatsTests {
+
+
+    @Test
+    public void entitiesCountTest() {
```

Comment:
Mir ist gerade erst aufgefallen, dass du hier nur das zählen mit coutEntities zählst. Ein anderer Teil wäre natürlich auch `com.bakdata.conquery.models.datasets.concepts.MatchingStats.Entry#addEvent` zu testen. Ich weiß aber nicht ob das so dringend ist, da es ja im MetaDataCollectionTest gemacht wird. Wie siehst du das @thoniTUB

Reply:
ja daran hatte ich gedacht, jedoch muss ich dafür noch Test-Table und Test-Bucket konstruieren können, wozu ich nicht in der Lage war. Wie kann ich das eurer Meinung nach realisieren?

Reply:
Ich hatte mit Brandon erstmal besprochen, dass wir den Test klein halten, damit es sich auf seine Implementierung konzentriert und es läuft.
Den MetadataCollectionTest hatte ich leider übersehen. Den könnte man aber auch leicht um einen Check für dieses Feature implementieren.
Den Unit-Test finde ich aber sinnvoll.

Reply:
@Njimefo falls du einen Test-Bucket erstellen willst, dann hilft dir das vielleicht: https://github.com/bakdata/conquery/blob/97d269544120a54c67f7111271c2d0904ba95cb7/backend/src/test/java/com/bakdata/conquery/models/events/CBlockTest.java#L43

Wir können aber gerne nochmal die generelle Struktur der Buckets durchsprechen/aufzeichenen.

Reply:
Ich hatte gestern mit ihm drüber geschaut, wenn er die Table ohne Columns hält, braucht er keinen Bucket.